### PR TITLE
Added Feature: tqdm progress bar disable func.

### PR DESCRIPTION
**Pull request by us**
Related: Option to turn tqdm progress bar off #310

### CODE REVIEW

**Comment by Conchylicultor** [comment]
This won't disable the download bar globally, no ? Only for download, but not when shuffling,...
Is what you wanted ?

**Comment by us** [comment]
Yes I wanted that because I understood the feature request like that. So I disable just tqdm progress bar.
Maybe we should ask @sfujiwara.

**Comment by sfujiwara** [comment]
In some environment such as Google Cloud ML Engine, it seems that tqdm (including shuffling) occurs an error and does not work.
Thus, I think wee need a feature to globally disable tqdm.

File: tensorflow_datasets/core/utils/tqdm.py [hunk 1]
```diff
@@ -59,28 +76,32 @@ class _TqdmPbarAsync(object):
...
+    self._lock = tqdm_lib.tqdm.get_lock()
     self._pbar = pbar
     self._tqdm_bars.append(pbar)
 
   def update_total(self, n=1):
     """Increment total pbar value."""
-    with self._lock:
-      self._pbar.total += n
-      self.refresh()
+    if active:
```
Comment (us) [inline tensorflow_datasets/core/utils/tqdm.py]: @Conchylicultor these `if statements` was little bad, how can I solve that ?

**Comment by Conchylicultor** [comment]
Thanks for your contribution. Merged it even if I slightly modified the implementation.

**Pull request merged**